cornell-dti/course-plan
# [EZ] 2/n Accessibility - Page regions

### Summary
Added page regions
![image](https://user-images.githubusercontent.com/55263191/111380117-524f6880-867a-11eb-8b18-c5371db31af4.png)


### Test Plan
- Make sure frontend still looks fine
- Make sure that the page regions I added make sense (refer to above picture)
